src/Sylius/Bundle/CoreBundle/Tests/DependencyInjection/Compiler/CheckStatisticsOrdersTotalsProviderTypePassTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Tests\DependencyInjection\Compiler;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Sylius\Bundle\CoreBundle\DependencyInjection\Compiler\CheckStatisticsOrdersTotalsProviderTypePass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+final class CheckStatisticsOrdersTotalsProviderTypePassTest extends AbstractCompilerPassTestCase
+{
+    /** @test */
+    public function it_passes_when_all_providers_are_present(): void
+    {
+        $this->setParameter('sylius_core.orders_statistics.intervals_map', ['daily' => 'Daily', 'monthly' => 'Monthly']);
+        $this->setDefinition(
+            'sylius.registry.statistics_orders_totals_provider',
+            (new Definition())
+                ->addTag('sylius.statistics.orders_totals_provider', ['type' => 'daily'])
+                ->addTag('sylius.statistics.orders_totals_provider', ['type' => 'monthly']),
+        );
+
+        $this->compile();
+        $this->assertTrue(true, 'Compiler pass completed without throwing an exception.');
```

Comment:
```suggestion
```
Makes no sense; Guessing it's only here to shut up unit throwing hissy fits about a risky test or some other crap. Just assert after compilation if the service and parameter are there.